support-frontend/app/wiring/Controllers.scala

```diff
@@ -33,6 +33,7 @@ trait Controllers {
     allSettingsProvider,
     appConfig.guardianDomain,
     appConfig.stage,
+    appConfig,
```

Comment:
Can you just pass the url in here please as you have for the other 3 controllers below

Reply:
that would make sense yes :)